### PR TITLE
Draft docs for the AI functions

### PR DESCRIPTION
**Pull request by eolivelli**
Summary:
- add a new minimal file for the ai-tools steps
- add some docs about how to configure "openai" and "datasource"

### CODE REVIEW

File: pulsar-ai-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+## Streaming Generative AI Extensions
+
+This module is an extensions of the standard Transformation function,
+and add some useful functions for Streaming generative AI.
+
+Currently available GenAI operations are:
+* [compute-ai-embeddings](#compute-ai-embeddings): computes an Embeddings vector for a given text.
+* [query](#qyery): queries a database and enrich the message with the results.
```
Comment (cbornet) [inline pulsar-ai-tools/README.md:8]:
```suggestion
* [query](#query): queries a database and enrich the message with the results.
```

File: pulsar-ai-tools/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Computes an embeddings vector for a given text. The embeddings vector is added to the message.
+
+Parameters:
+
+| Name             | Description                                                                                                         |
+|------------------|---------------------------------------------------------------------------------------------------------------------|
+| model            | The model to use, it depends on the provider                                                                        |
+| compute-service  | The embeddings provider (openai, huggingface...)                                                                    | 
+| text             | Template for the text to compute the embeddings on. You can use the Mustache syntax (for example {{value.field1}} ) |
```
Comment (cbornet) [inline pulsar-ai-tools/README.md:24]: Ad link to mustache spec ?

File: pulsar-ai-tools/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| output-field      | The name of the field to add or update (for example value.embeddingsvector)                                                                                                                                                                 |
+| messages          | Template for the messages, it is an array. You can use Mustache in order to fill in the template with the fields of the message.                                                                                                            |
+| model             | ID of the model to use. See the [model endpoint compatibility](https://platform.openai.com/docs/models/model-endpoint-compatibility) table for details on which models work with the Chat API.                                              |
```
Comment (cbornet) [inline pulsar-ai-tools/README.md:55]: Models have different names on Azure. We could also put a link for Azure ?

**Review by cbornet — APPROVED** [review]

File: pulsar-ai-tools/README.md (hunk 3 above)
Comment (eolivelli) [reply pulsar-ai-tools/README.md:55]: good idea

File: pulsar-ai-tools/README.md (hunk 2 above)
Comment (eolivelli) [reply pulsar-ai-tools/README.md:24]: good idea

**Pull request merged**